src/main/java/uk/gov/hmcts/reform/ccd/client/CoreCaseDataApi.java

```diff
@@ -69,6 +70,31 @@ CaseDetails submitForCaseworker(
             @RequestParam Map<String, Object> searchCriteria
     );
 
+    @RequestMapping(
+            method = RequestMethod.GET,
+            value = "/caseworkers/{userId}/jurisdictions/{jurisdictionId}/case-types/{caseType}/cases/pagination_metadata",
+            headers = CONTENT_TYPE + "=" + APPLICATION_JSON_VALUE)
+    PaginatedSearchMetadata searchCasesMetadataForCaseworkers(@RequestHeader(AUTHORIZATION) String authorisation,
```

Comment:
does this mean the existing search endpoint is deprecated? Looks like this is a replacement? or is it not?

Reply:
It's not, this endpoint only gives you pagination details: how many results and how many pages there are for the current search.

However, I do agree that `searchCasesMetadata` is confusing as we do have a search feature on case metadata. Maybe something like `getPaginationInfoForSearch` would make more sense?

Reply:
ok. Changed to
getPaginationInfoForSearchForCaseworkers
getPaginationInfoForSearchForCitizens